src/System.Diagnostics.Process/src/System/Diagnostics/ProcessStartInfo.Unix.cs

```diff
@@ -16,20 +16,20 @@ public sealed partial class ProcessStartInfo
 
         public string PasswordInClearText
         {
-            get { throw new PlatformNotSupportedException(SR.ProcessStartIdentityNotSupported); }
-            set { throw new PlatformNotSupportedException(SR.ProcessStartIdentityNotSupported); }
+            get { throw new PlatformNotSupportedException(string.Format(SR.ProcessStartSingleFeatureNotSupported, nameof(PasswordInClearText))); }
```

Comment:
Sorry to be late to the party, but we generally do `SR.Format` instead of `string.Format` when operating on something out of SR (specifically if it's an exception message)

The reason I guess is explained here https://github.com/dotnet/corefx/blob/master/src/Common/src/System/SR.cs#L17

Basically when building for a precompiled platform (such as .NET Native) the exception messages may be stripped or mangled for space. But they want to keep the parameter values so the compiler force set `UsingResourceKeys` to true and that will cause it to just suffix them instead of do string.format (which would not work because the replacement markers are lost)

I don't fully understand how this could work though probably because I don't know exactly what the .NET Native compiler does. @jkotas do we need to care about this?

Reply:
@jkotas I guess I was curious how it knows to drop only exception messages from the resources, and not other random messages. And also why it doesn't just rewrite them leaving the replacement markers, so that string.format would still work.